### ISSUE TITLE
TestNet server broadcust not working

### ISSUE THREAD

**Issue by e1ektr0** [MetacoSA/QBitNinja#53]
I done successful broadcast, but looks like, it's not working. Can you check it?

Tx hash:fdf5117b6820a39f9c022fe01609ed111ce99bf72ed92be103ed21a2f4ba266b

**Comment by NicolasDorier** [comment]
worked https://testnet.smartbit.com.au/tx/fdf5117b6820a39f9c022fe01609ed111ce99bf72ed92be103ed21a2f4ba266b

**Comment by e1ektr0** [comment]
Yes, it's working. It's start working 1hr later after i broadcast it.

but. some times. for example right now. i broadcast some tx(example:0615d30dbf5d34af0cf0c4e399923f86bb59edda315fe397688740c9f1313adc), then i request it from qbit server. And it's return null. It's very strange. I broadcast it on this server, but it don't know about it tx. How it possible?

**Comment by e1ektr0** [comment]
Also. I provide extra fee. No effect.
Check last processed block by qbit server - it's fine. Only broadcast not working.

**Comment by NicolasDorier** [comment]
> but. some times. for example right now. i broadcast some tx(example:0615d30dbf5d34af0cf0c4e399923f86bb59edda315fe397688740c9f1313adc), then i request it from qbit server. And it's return null

Possible if not enough fees.

**Comment by e1ektr0** [comment]
Sorry, but no. 

1)I add a lot of extra fee.
2)I publish same raw tx via 3 party service (via http ui), and it was published in one seccond. And qbit server get this tx.
